src/main/docker/logback.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright 2017-2020 Micro Focus or one of its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<configuration>  
+
+    <appender name="STDOUT" class="com.github.cafapi.logging.tomcat.juli.CAFTomcatConsoleAppender">
```

Comment:
Why did you introduce this instead of using `CAFConsoleAppender` which looks identical?

Reply:
org.apache.juli.logging.ch.qos.logback.classic.spi.LoggingEvent, org.apache.juli.logging.ch.qos.logback.core.ConsoleAppender these two classes are different in logback with new drop in. If i try to use CAFConsoleAppender I am getting below exception
11:44:19.923 documentid-service> 11:44:19,914 |-ERROR in ch.qos.logback.core.joran.action.AppenderAction - Could not create an Appender of type [com.github.cafapi.logging.logback.CAFConsoleAppender]. ch.qos.logback.core.util.IncompatibleClassException
11:44:19.923 documentid-service> 	at ch.qos.logback.core.util.IncompatibleClassException
11:44:19.923 documentid-service> 	at 	at ch.qos.logback.core.util.OptionHelper.instantiateByClassNameAndParameter(OptionHelper.java:58)
11:44:19.923 documentid-service> 	at 	at ch.qos.logback.core.util.OptionHelper.instantiateByClassName(OptionHelper.java:45)
11:44:19.923 documentid-service> 	at 	at ch.qos.logback.core.util.OptionHelper.instantiateByClassName(OptionHelper.java:34)

In CAFConsoleAppender LoggingEvent and ConsoleAppender are from different packages.

My local repo had some issues. Raising new PR for this change in the new Branch.